docs/en/introduction.md

```diff
@@ -0,0 +1,99 @@
+# Introduction
+
+> In this chapter, we will use the Ribir syntax to write some simple examples. You only need to understand the general idea, no need to go into details. We will introduce it in detail in the following chapters.
+
+## What is Ribir?
+
+Ribir is a Rust framework for building cross-platform user interfaces. It allows you to compile applications for desktop, mobile and browser with one code. Ribir uses a non-intrusive declarative programming model that allows you to develop and design user interfaces as an independent module.
+
+Its core design concept is:
+
+> The UI is a re-description of data interaction and continues to respond to modifications of data.
+
+We emphasize "re" here because we think the API is the first description of data interaction, and building UI with Ribir requires only the data API.
```

Comment:
The "re" here signifies that the API serves as the initial description of the data interaction. When building with Ribir, developers only need to focus on the data API to create the UI.